docs/articles/documentation/reference/testcafe-api/browserconnection/opened.md

```diff
@@ -0,0 +1,18 @@
+---
+layout: docs
+title: BrowserConnection.opened Event
+permalink: /documentation/reference/testcafe-api/browserconnection/opened.html
+---
+# BrowserConnection.opened Event
+
+Fires when a browser launched by a [browser provider](../../../guides/extend-testcafe/browser-provider-plugin.md#implement-the-browser-provider) has connected to TestCafe and opened a test page.
```

Comment:
> and opened a test page

I'm not sure about that. It's better to say that a browser is ready to run tests or smth like that.

Reply:
> ...has connected to TestCafe and reported that it is ready to run tests.